features/pull_request.feature

```diff
@@ -160,6 +160,41 @@ Feature: hub pull-request
     When I successfully run `hub pull-request`
     Then the output should contain exactly "the://url\n"
 
+  Scenario: Single-commit pull request with "--no-edit"
+    Given the GitHub API server:
+      """
+      post('/repos/mislav/coral/pulls') {
+        assert :title => 'Commit message 1',
+               :body => nil
```

Comment:
Can the test also verify that the pull request body is obtained from the git commit message as well?